### DISCUSSION TITLE
Privilege Cloud PVWA address

### DISCUSSION THREAD

**Discussion by Dana0619** [cyberark/epv-api-scripts#362] [General]
Greetings,

I have opened a ticket with Cyberark to get the address of our Privileged Cloud PVWA url but I cant get it to resolve or connect.

If someone else has done this with Privilege Cloud could you post the format of the URL pls.

Here is what I have.

"https://vault-<mycompany>.privilegecloud.cyberark.cloud/PasswordVault"

Thank you

**Comment by Jflesc01** [comment]
Hello,
To find the tenant URL, in the Identity Administration portal, go to Settings > Customizations > Tenant URLs. To find the tenant ID, click the user icon in the top-right corner, then click About.

https://docs.cyberark.com/Idaptive/Latest/en/Content/GetStarted/tenant-url-domains.htm?TocPath=Get%20Started%7C_____6#:~:text=To%20find%20the%20tenant%20URL,Settings%20%3E%20Customizations%20%3E%20Tenant%20URLs
